tendrl/ceph_integration/manager/rpc.py

```diff
@@ -20,13 +20,14 @@
 
 class EtcdRPC(object):
 
-    def __init__(self, manager):
+    def __init__(self, manager, Etcdthread):
```

Comment:
why is this change required here, is the same required in tendrl/gluster_integration/manager?